### ISSUE TITLE
running to an error exec format error

### ISSUE THREAD

**Issue by njinco** [samdbmg/dhcp-netboot.xyz#14]
your setup is the closes i want to  get for not wanting to touch my router as my dhcp server. but i get this error from running this. Any tips?

root@ubuntu:/home/ubuntu# docker run --net=host --cap-add=NET_ADMIN -e DHCP_RANGE_START=192.168.1.1 samdbmg/dhcp-netboot.xyz
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
exec /init: exec format error
root@ubuntu:/home/ubuntu# docker run --platform linux/amd64 --net=host --cap-add=NET_ADMIN -e DHCP_RANGE_START=192.168.1.1 samdbmg/dhcp-netboot.xyz
exec /init: exec format error

**Comment by samdbmg** [comment]
Are you running this on a Raspberry Pi or something? That'll be because I haven't built this for the ARM architecture I'm afraid.

For now, if you clone the repo and do `docker build -t local/dhcp-netboot.xyz .`, then change your command above to `docker run --net=host --cap-add=NET_ADMIN -e DHCP_RANGE_START=192.168.1.1 local/dhcp-netboot.xyz` that should work, although I don't actually have an ARM system I can test that on.

I better go off and learn about Docker multi-arch manifests to see if I can make CI do that, but it might be a while before I get to it

**Comment by njinco** [comment]
yes i am trying to run on a raspberry pi 4. But i thought i address that via --platform linux/amd64 command. but im not the good with docker and i will try your suggestion after i get some free time. I will let you know if it works. Thanks Sam for the reply.
